src/LanguageServer/Impl/Implementation/Server.Completion.cs

```diff
@@ -146,6 +146,7 @@ await InvokeExtensionsAsync((ext, token)
                     filterText = x.filterText,
                     preselect = x.preselect,
                     insertText = x.insertText,
+                    insertTextFormat = (PythonTools.Analysis.LanguageServer.InsertTextFormat)x.insertTextFormat,
```

Comment:
using? Definitions that are in the engine are obsolete and are kept around for backward compat. POR to remove them in a month or so.

Reply:
I'm not sure if any of that is used, but I wanted to be thorough and make sure it's set anywhere it can be. Once all of the old types go away this'll probably also be deleted (and we can probably start setting sane defaults too).